app/forms/chargeA/ChargeDetailsFormProvider.scala

```diff
@@ -21,8 +21,15 @@ import javax.inject.Inject
 import models.chargeA.ChargeDetails
 import play.api.data.Form
 import play.api.data.Forms.mapping
+import uk.gov.voa.play.form.Condition
+import uk.gov.voa.play.form.ConditionalMappings._
 
 class ChargeDetailsFormProvider @Inject() extends Mappings with Constraints {
+  private def condition(field: String): Condition =
```

Comment:
Maybe call this something more meaningful, e.g. isEmptyOrBothHaveValue

Reply:
Done